exercises/simple-cipher/tests/simple-cipher.rs

```diff
@@ -0,0 +1,136 @@
+extern crate simple_cipher;
+use simple_cipher::*;
+use std::collections::HashSet;
+
+const PLAIN_TEXT: &str = "thisismysecret";
+const KEY: &str = "abcdefghij";
+
+
+#[test]
+fn cipher_can_encode_with_given_key() {
+    assert_eq!(encode(KEY, "aaaaaaaaaa"), Some(KEY.to_string()));
+}
+
+#[test]
+#[ignore]
+fn cipher_can_decode_with_given_key() {
+    assert_eq!(decode(KEY, "abcdefghij"), Some("aaaaaaaaaa".to_string()));
+}
+
+#[test]
+#[ignore]
+fn cipher_is_reversible_given_key() {
```

Comment:
It does not make sense for me to request this to change here, since this was just taken from other tracks.

Please excuse me as I use this PR as a staging ground for comments I am to make to the problem-specifications PR adding canonical tests.

This test is only a demonstration of possibilities, and has no power to reject any implementations that the previous tests have not already rejected.

Is the advantage of demonstrating possibilities worth having the test?

I would say it is not.